### DISCUSSION TITLE
Best for current conditions

### DISCUSSION THREAD

**Discussion by jfbauer432** [weather-gov/api#346] [Q&A] [answered]
I am currently using https://api.weather.gov/stations/STATION/observations/latest for current conditions.  However, they seem to update only about once over couple of hours and sometimes a lot longer.  Not the most current.

What is the best way to get current conditions for a location?  i.e. temp, wind chill / heat index, humidity, pressure, and something that can be mapped to an icon.  Ideally data more then about 1 hour old.

**Comment by scadergit** [comment, marked as answer]
The /latest endpoint is the latest observation in the database. There is a known issue where observations are slow to ingest into the system. Please [open an opterational issue](https://weather-gov.github.io/api/reporting-issues) if you are not getting the observation expected from that endpoint.

**Comment by jfbauer432** [comment]
There already was a ticket (INC0219108).  Looks like it has been resolved.  I am seeing station data that is about 1h18m to 1h27m old currently.  It would be nice it it was shorter, but at leat it is back to normal.

Description of resolution:

> *2020-05-13 19:08:40 - John Percy*^Additional comments
> As of, 1845z, A file purge was completed to resolve an issue where certain 
> processes were "stuck" due to some files that were corrupt, or improperly 
> formatted and causing problems.
> 
> Please advise if you find a restoration to the "norm" you have come to expect, 
> or if you find issues continue.

**Comment by scadergit** [comment]
Fair expectation. There are complications beyond our control that have increased observation ingest latency. We are working with NCO and the upstream provider to resolve them. Some efforts over the next week might slightly improve latency, but we won't see the ideal goal (less than 20m) for a while.
